modules/book.rb

```diff
@@ -0,0 +1,37 @@
+require 'json'
+require_relative './book'
+
+module PreserveBook
+  FILE_NAME = './database/book.json'.freeze
+
+  def create_book_class(arr)
+    new_arr = []
+    arr.each do |el|
+      new_arr << Book.new(el['title'], el['author'])
+    end
+
+    new_arr
+  end
+
+  private :create_book_class
+
+  # we need to open the file
+  def fetch_books
+    File.new(FILE_NAME.to_s, 'w') unless File.exist?(FILE_NAME)
+    file = File.read(FILE_NAME)
+    data = file.empty? ? [] : JSON.parse(file)
+    create_book_class(data)
+  end
+
+  # we can write to the file
+  def preserve_books(data)
+    new_data = []
+    data.each do |d|
+      new_data << {
+        title: d.title,
+        author: d.author
+      }
+    end
+    File.write(FILE_NAME, JSON.generate(new_data))
+  end
```

Comment:
- [ ] Great job working on the functionality so far but I think it is not working properly so please fix the issue so I can be able to see the data preserved in files since after running the app and entering some data then exit the app I can't see the data that I entered is displayed there although I tried after deleing the files to see it just created the files to me but nothing saved! 💯✨🤩 so kindly make sure that is working fine 💯✅✅


## I created a person but take a look at the file is empty ❗❗ and it is the same for a book and a rental ✅✅
![image](https://user-images.githubusercontent.com/58553711/226769832-d38bf33e-db47-4c25-b398-f2879e662945.png)